src/hyperion/__main__.py

```diff
@@ -58,17 +58,15 @@ def __init__(self, exception: Exception) -> None:
 
 
 class BlueskyRunner:
-    command_queue: "Queue[Command]" = Queue()
-    current_status: StatusAndMessage = StatusAndMessage(Status.IDLE)
-    last_run_aborted: bool = False
-    aperture_change_callback = ApertureChangeCallback()
-    RE: RunEngine
-    skip_startup_connection: bool
-    context: BlueskyContext
-
```

Comment:
Were these originally class attributes because we assume that there would never be more than one `BlueskyRunner`?

Reply:
yes, I guess, or we just didn't know how python worked. we still think there's only one BlueskyRunner in real life, but in tests they seem to not always get garbage collected quickly enough